content/es/docs/contribute/localization_es.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Contribuir a la documentación de Kubernetes en español
+content_type: concept
+---
+
+<!-- overview -->
+
+¡Bienvenida/o! Esta página es un esbozo.
+
+
+<!-- body -->
+
+## Glosario de terminología {#terminologia}
+
+|      English      |        Español         |  Género   |    Commentarios          |
+| ----------------- | ---------------------- | --------- | ------------------------ |
+| availability zone | zona de disponibilidad |           |                          |
+| bearer token      | ⚠ ?                    |           |                          |
+| built-in          | incorporados           |           |                          |
+| conditions        | estado                 |           | para node conditions     |
+| container         | contendador            |           |                          |
+| controller        | controlador            |           |                          |
+| deploy            | desplegar              |           |                          |
+| Deployment        | Deployment             |           | objeto Kubernetes        |
+| Endpoints         | Endpoints              |           | objeto Kubernetes        |
+| file              | fichero                |           |                          |
```

Comment:
This word is translated as `fichero` mostly in Spain but in Latin america is used `archivo`

Reply:
Solved in https://github.com/kubernetes/website/pull/24477/commits/c1aa3cc7a1a249bf0a7f94a99d6963cf4766a65e